articles/service-fabric/service-fabric-backuprestoreservice-configure-periodic-backup.md

```diff
@@ -79,7 +79,32 @@ A backup policy consists of the following configurations:
             ```
 
 * **Backup storage**: Specifies the location to upload backups. Storage can be either Azure blob store or file share.
-    1. **Azure blob store**: This storage type should be selected when the need is to store generated backups in Azure. Both _standalone_ and _Azure-based_ clusters can use this storage type. Description for this storage type requires connection string and name of the container where backups need to be uploaded. If the container with the specified name is not available, then it gets created during upload of a backup.
+    1. **Azure blob store with managed identity**: This storage type should be selected when the need is to store generated backups in Azure. Both _standalone_ and _Azure-based_ clusters can use this storage type. Description for this storage type requires BlobServiceUri and name of the container where backups need to be uploaded. If the container with the specified name is not available, then it gets created during upload of a backup. Replace `account-name` with your storage account name.
```

Comment:
`_Azure-based_` should be generalized to cloud based clusters as SF clusters can be created on any cloud